application/qt_plot.py

```diff
@@ -84,12 +77,12 @@ def draw(self):
 
 
 def plot_results(main_window):
-    if ac.CURRENT_PROJECT_PATH:
-        database_filename = os.path.join(ac.CURRENT_PROJECT_PATH, "run", "results.sqlite")
-        homography_filename = os.path.join(ac.CURRENT_PROJECT_PATH, "homography", "homography.txt")
-        camera_image = os.path.join(ac.CURRENT_PROJECT_PATH, "homography", "camera.png")
-        fps_exists, video_fps = check_project_cfg_option("video", "framerate")
-        video_fps = float(video_fps)
+    project_path = get_project_path()
+    if project_path:
+        database_filename = os.path.join(project_path, "run", "results.sqlite")
+        homography_filename = os.path.join(project_path, "homography", "homography.txt")
+        camera_image = os.path.join(project_path, "homography", "camera.png")
+        video_fps = 30 # TODO: What?
```

Comment:
`def get_framerate(videopath):`
`    list_o = str(subprocess.check_output(["ffprobe",`
`       "-v", "error", `
`       "-select_streams", "v:0", `
`       "-show_entries", "stream=avg_frame_rate", `
`       "-of", "default=noprint_wrappers=1:nokey=1", `
`        videopath]))`
`    return str(int(list_o.strip().split('/')[0])/float(list_o.strip().split('/')[1]))`